tests/release_automation/docker/executor/Dockerfile

```diff
@@ -23,11 +24,23 @@ RUN curl -O -J https://dl.cloudsmith.io/public/ebivariation/packages/maven/uk/ac
 RUN cp eva-accession-release-*exec.jar /usr/local/software/eva-accession-release.jar
 
 
+# install vcf_validator
+RUN curl -LJo /usr/local/bin/vcf_validator https://github.com/EBIvariation/vcf-validator/releases/download/v${VCF_VALIDATOR_VERSION}/vcf_validator_linux \
+    && chmod 755 /usr/local/bin/vcf_validator
+
+# install vcf_assembly_checker
+RUN curl -LJo /usr/local/bin/vcf_assembly_checker https://github.com/EBIvariation/vcf-validator/releases/download/v${VCF_VALIDATOR_VERSION}/vcf_assembly_checker_linux \
+    && chmod 755 /usr/local/bin/vcf_assembly_checker
+
+# Install contig_name_converter
+#RUN pip install -q "git+https://github.com/EBIvariation/contig_name_converter.git@${CONTIG_NAME_CONVERTER_VERSION}"
+RUN pip install -q "git+https://github.com/tcezard/contig_name_converter.git@EVA3793-allow-accession-as-id"
```

Comment:
Will revert to using the released version once https://github.com/EBIvariation/contig-name-converter/pull/6 is merged and released